packages/aws-cdk-lib/aws-appsync/lib/graphqlapi.ts

```diff
@@ -442,6 +442,20 @@ export interface GraphqlApiProps {
    * @default IntrospectionConfig.ENABLED
    */
   readonly introspectionConfig?: IntrospectionConfig;
+
```

Comment:
both of these new props have lower and upper bounds according to the docs: https://docs.aws.amazon.com/appsync/latest/APIReference/API_GraphqlApi.html

They should be documented in their docstrings, checked for at synth time, and also have a unit test for each testing the boundary

Reply:
Added